README.md

```diff
@@ -22,8 +22,9 @@ Monorepo containing all the main components of Streamr Network.
 * [cli-tools](packages/cli-tools/README.md) (@streamr/cli-tools)
 
 ### Internal
-* [network-node](packages/network/README.md) (@streamr/network-node)
-* [network-tracker](packages/network-tracker/README.md) (@streamr/network-tracker)
+* [trackerless-network](packages/trackerless-network/README.md) (@streamr/trackerless-network)
```

Comment:
Broken link: could add README to trackeless-network

Reply:
Added placeholder README-file